propulsion_motor/platformio.ini

```diff
@@ -8,17 +8,25 @@
 ; Please visit documentation for the other options and examples
 ; https://docs.platformio.org/page/projectconf.html
 
-[env:denky32]
-platform = https://github.com/Jason2866/platform-espressif32.git
-board = nodemcu-32s
+[env:esp32-s3-devkitc-1]
+platform = espressif32
+board = esp32-s3-devkitc-1
 framework = arduino
+
 monitor_speed = 115200
 monitor_raw = true
 
 lib_deps = 
     rover_can_lib=symlink://../lib/rover_can_lib
     lib_rover=symlink://../lib/lib_rover
 
-build_flags = 
-	'-D VERBOSE=0'
+build_unflags =
+    -w CCFLAGS
+    -std=gnu++11
+
+build_flags =
+    '-D ARDUINO_USB_CDC_ON_BOOT=1'
+    '-D ARDUINO_USB_MODE=1'
+
+    '-D VERBOSE'
     '-D LOGGER_LOWEST_LEVEL=0'
```

Comment:
Update documentation if needed in REAMDE

Reply:
flemme, but yes, done